spec/models/post_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  subject(:post) { Post.create(title: 'Test post', comments_counter: 1, likes_counter: 20) }
+
+  it 'should have a title' do
+    post.title = nil
+    expect(post).to_not be_valid
+  end
+
+  it 'should have a valid length of title' do
+    post.title = 'a' * 255
+    expect(post).to_not be_valid
+  end
+
+  it 'comments counter should be intger' do
+    post.comments_counter = 'a'
+    expect(post).to_not be_valid
+  end
+
+  it 'should have a comments counter' do
+    post.comments_counter = nil
+    expect(post).to_not be_valid
+  end
+
+  it 'comments counter should be greater than or equal to 0' do
```

Comment:
It says in the requirements that you are required to have unit tests for posts, users, comments and likes. You only have unit tests for posts and users. It would be nice if added unit tests for comments and likes.

![requirements](https://user-images.githubusercontent.com/79506105/206768790-0e341148-1757-43d8-8e14-7f1562b3579a.PNG)